src/writer/proposal.ts

```diff
@@ -141,8 +141,20 @@ export async function verify(body): Promise<any> {
       }
 
       if (!isValid) return Promise.reject('validation failed');
-    } catch (e) {
-      capture(e, { space: msg.space, address: body.address });
+    } catch (e: any) {
+      if (e instanceof Error) {
+        capture(e, { space: msg.space, address: body.address });
+      } else {
+        capture(new Error('Validation failed'), {
+          contexts: {
+            validationError: e,
+            input: {
+              space: msg.space,
+              address: body.address
+            }
+          }
+        });
```

Comment:
Could we handle this in `capture` itself? maybe a new params `options` with something like `defaultError` inside it,

Reply:
Sounds like a good idea 👍